javascript/modules/client/README.md

```diff
@@ -431,33 +431,33 @@ console.log(proposals);
     voters: [
       {
         address: "0x8367dc645e31321CeF3EeD91a10a5b7077e21f70",
-        voteValue: VoteOptions.YEA,
+        voteValue: VoteOptions.YES,
         weight: 1,
       },
       {...}
     ],
   },] */
 ```
 
-## Multisig governance plugin client
+## AllowList governance plugin client
```

Comment:
We need to populate the Readme with the new method/encoder/estimator examples.

We also need to update the changelos (Subgraph and mocked methods)